<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFluid</class>
 <widget class="QWidget" name="WidgetFluid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravity</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxGravity">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>9.810000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Density</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDensity">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxViscosity">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Viscosity</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxStiffness">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Stiffness</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRadius">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>2.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Particle Mass</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>2.000000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
